rt/src/test/java/io/fabric8/maven/rt/SpringbootHttpBoosterIT.java

```diff
@@ -16,19 +16,16 @@
 
 package io.fabric8.maven.rt;
 
-import io.fabric8.openshift.api.model.*;
+import io.fabric8.openshift.api.model.Route;
+import java.util.concurrent.TimeUnit;
 import org.apache.http.HttpStatus;
 import org.eclipse.jgit.lib.Repository;
 import org.junit.After;
-import org.testng.annotations.Test;
-
-import java.util.concurrent.TimeUnit;
+import org.junit.Test;
 
 import static io.fabric8.kubernetes.assertions.Assertions.assertThat;
 
-/**
- * Created by hshinde on 11/23/17.
- */
+
```

Comment:
Any specific reason to remove this comment? I think it's perfectly fine to have this.

Reply:
I cannot see it for any other test under `rt/`. I removed it to make it consistent across all tests.

Reply:
Hm, okay. I also don't have any problems with it being gone :-)